### DISCUSSION TITLE
Support ensemble in RIFE?

### DISCUSSION THREAD

**Discussion by banjaminicc** [AmusementClub/vs-mlrt#24] [General]
Running the 4.x models with ensemble is known to provide better results(usually with less warping) at the cost of speed and can be quite useful in certain scenarios
Is it possible to implement it in vsmlrt.py or will we have to build our own model then run it with vsmlrt?

**Comment by WolframRhodium** [comment]
The implementation is possible. We want to achieve a balance between many high-quality built-in models and binary size, and we are busy on other projects right now, so this may not be currently implemented.

**Comment by WolframRhodium** [comment]
done, you need to update `vsmlrt.py` and download ensemble models [here](https://github.com/AmusementClub/vs-mlrt/releases/download/external-models/rife_ensemble_v1.7z)

**Comment by banjaminicc** [comment]
Thanks for the insanely quick update haha
cheers!
